model/metrics/system-metrics.yaml

```diff
@@ -105,12 +103,24 @@ groups:
   - id: metric.system.memory.usage
     type: metric
     metric_name: system.memory.usage
-    brief: ""
+    brief: "Reports memory in use by state."
+    note: |
+      The sum over all `system.memory.state` values SHOULD equal the total memory
+      available on the system, that is `system.memory.limit`.
     instrument: updowncounter
     unit: "By"
     attributes:
       - ref: system.memory.state
 
+  - id: metric.system.memory.limit
+    type: metric
+    metric_name: system.memory.limit
+    brief: "Total memory available in the system."
+    note: |
```

Comment:
Slipped my mind, but can't we specify the requirement level here? I think the ones we have today are all recommended so that's why it's not listed, But this new seems to be Opt-in.

Reply:
How should I do that? I see some other metrics that are labeled as optional or opt-in but the requirement level is not specified in the YAML:
- `http.server.active_requests` is listed as optional [in the docs](https://github.com/open-telemetry/semantic-conventions/blob/5aa01542b97e73a0c48a60bd71d762017d0d65e1/docs/http/http-metrics.md#metric-httpserveractive_requests) but [not on the YAML](https://github.com/open-telemetry/semantic-conventions/blob/5aa01542b97e73a0c48a60bd71d762017d0d65e1/model/metrics/http.yaml#L55-L101)
- `jvm.system.cpu.utilization` is listed as opt-in [in the docs](https://github.com/open-telemetry/semantic-conventions/blob/5aa01542b97e73a0c48a60bd71d762017d0d65e1/docs/runtime/jvm-metrics.md#metric-jvmsystemcpuutilization) but also [not on the YAML](https://github.com/open-telemetry/semantic-conventions/blob/5aa01542b97e73a0c48a60bd71d762017d0d65e1/model/metrics/jvm-metrics-experimental.yaml#L10-L20)

I see `requirement_level: opt_in` in a bunch of places but only on (resource/span/metric) attributes.

Reply:
I get 
```
Error parsing /source/metrics/system-metrics.yaml

Invalid keys: ['requirement_level'] - @123:5
```

If I add this to the metric, so I don't think `semconvgen` supports this today.

Reply:
Alright, thanks for looking! I forgot this wasn't supported, I even opened an issue for it already 😅  https://github.com/open-telemetry/build-tools/issues/181. Sorry for making you do extra work.